include/envoy/http/filter.h

```diff
@@ -118,6 +119,12 @@ class StreamFilterCallbacks {
   virtual AccessLog::RequestInfo& requestInfo() PURE;
 
   /**
+   * @return span context used for tracing purposes. Individual filters may add or modify
+   * information in the span context.
```

Comment:
nit formatting: Have the word "information" start at the same space as "context" Ex:
https://github.com/lyft/envoy/blob/master/include/envoy/http/conn_pool.h#L58